gtk-2.yaml

```diff
@@ -1,5 +1,5 @@
 package:
-  name: gtk-2.0
+  name: gtk-2
```

Comment:
@Dentrax gtk-2.0 is not a version stream, but an obsolete fork of the project. All other distributions use "2.0" for the ancient fork, and "3" for the currentish fork. Thus my preference would be to leave this historical name as is.

Note, all subpackages are still called "2.0" - did you want to change all of those too? and thus all reverse depends?

Reply:
Ah TIL that, thank you! FWIW better keep this as `-2.0`.